manifest/x86_64/b/bind.filelist

```diff
@@ -259,60 +251,29 @@
 /usr/local/lib64/bind/filter-aaaa.so
 /usr/local/lib64/bind/filter-a.la
 /usr/local/lib64/bind/filter-a.so
-/usr/local/lib64/libbind9-9.19.9.so
+/usr/local/lib64/libbind9-9.18.16.so
 /usr/local/lib64/libbind9.la
 /usr/local/lib64/libbind9.so
-/usr/local/lib64/libdns-9.19.9.so
+/usr/local/lib64/libdns-9.18.16.so
 /usr/local/lib64/libdns.la
 /usr/local/lib64/libdns.so
-/usr/local/lib64/libirs-9.19.9.so
+/usr/local/lib64/libirs-9.18.16.so
 /usr/local/lib64/libirs.la
 /usr/local/lib64/libirs.so
-/usr/local/lib64/libisc-9.19.9.so
-/usr/local/lib64/libisccc-9.19.9.so
+/usr/local/lib64/libisc-9.18.16.so
+/usr/local/lib64/libisccc-9.18.16.so
 /usr/local/lib64/libisccc.la
 /usr/local/lib64/libisccc.so
-/usr/local/lib64/libisccfg-9.19.9.so
+/usr/local/lib64/libisccfg-9.18.16.so
 /usr/local/lib64/libisccfg.la
 /usr/local/lib64/libisccfg.so
 /usr/local/lib64/libisc.la
 /usr/local/lib64/libisc.so
-/usr/local/lib64/libns-9.19.9.so
+/usr/local/lib64/libns-9.18.16.so
 /usr/local/lib64/libns.la
 /usr/local/lib64/libns.so
 /usr/local/sbin/ddns-confgen
 /usr/local/sbin/named
 /usr/local/sbin/rndc
 /usr/local/sbin/rndc-confgen
 /usr/local/sbin/tsig-keygen
-/usr/local/share/man/man1/arpaname.1.zst
-/usr/local/share/man/man1/delv.1.zst
-/usr/local/share/man/man1/dig.1.zst
-/usr/local/share/man/man1/dnssec-cds.1.zst
-/usr/local/share/man/man1/dnssec-dsfromkey.1.zst
-/usr/local/share/man/man1/dnssec-importkey.1.zst
-/usr/local/share/man/man1/dnssec-keyfromlabel.1.zst
-/usr/local/share/man/man1/dnssec-keygen.1.zst
-/usr/local/share/man/man1/dnssec-revoke.1.zst
-/usr/local/share/man/man1/dnssec-settime.1.zst
-/usr/local/share/man/man1/dnssec-signzone.1.zst
-/usr/local/share/man/man1/dnssec-verify.1.zst
-/usr/local/share/man/man1/host.1.zst
-/usr/local/share/man/man1/mdig.1.zst
-/usr/local/share/man/man1/named-checkconf.1.zst
-/usr/local/share/man/man1/named-checkzone.1.zst
-/usr/local/share/man/man1/named-compilezone.1.zst
-/usr/local/share/man/man1/named-journalprint.1.zst
-/usr/local/share/man/man1/named-rrchecker.1.zst
-/usr/local/share/man/man1/nsec3hash.1.zst
-/usr/local/share/man/man1/nslookup.1.zst
-/usr/local/share/man/man1/nsupdate.1.zst
-/usr/local/share/man/man5/named.conf.5.zst
-/usr/local/share/man/man5/rndc.conf.5.zst
-/usr/local/share/man/man8/ddns-confgen.8.zst
-/usr/local/share/man/man8/filter-a.8.zst
-/usr/local/share/man/man8/filter-aaaa.8.zst
-/usr/local/share/man/man8/named.8.zst
-/usr/local/share/man/man8/rndc.8.zst
-/usr/local/share/man/man8/rndc-confgen.8.zst
-/usr/local/share/man/man8/tsig-keygen.8.zst
```

Comment:
What happened to the manpages?

Reply:
I just changed the version numbers and built...
Are those extra man pages from the newer version that got removed? Or all of them?

Reply:
I have no idea.  Might need to include a switch option for the manpages in the stable release.

Reply:
Fixed.